jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -413,30 +474,47 @@ def _preset_args_changed(self, msg={}):
             # https://github.com/spacetelescope/jwst_novt/blob/main/jwst_novt/interact/display.py
             new_marks = []
             for i, reg in enumerate(regs):
+                if (not isinstance(reg, regions.Region)
+                        and not hasattr(reg, 'to_pixel')):  # pragma: nocover

```

Comment:
```suggestion
                        or not hasattr(reg, 'to_pixel')):   # pragma: no cover
```

Reply:
good catch!  I switched this logic around so many times I got lost 😵‍💫